packages/tomls/src/oracle_manager/configs/storkMark/aave_usdc.toml

```diff
@@ -0,0 +1,26 @@
+[invoke.oracle_manager_register_node_aave_usdc_mark_stork]
+target = ["OracleManagerProxy"]
+fromCall.func = "owner"
+func = "registerNode"
+args = [
+    "<%= settings.ORACLE_MANAGER_NODE_TYPE_DIV_REDUCER %>",
+    "0x",
+    "<%= defaultAbiCoder.encode(['address','string'], [contracts.OracleAdaptersProxy.address, settings.storkAssetPairIdAaveUSDCMark]) %>",
+    [
+        "<%= settings.aaveUsdMarkNodeIdStork %>",
+        "<%= settings.usdcUsdNodeIdStork %>",
```

Comment:
future prices do not apply to usdc-usd pair right?

Reply:
we normalise using the spot, following stork methodology